src/client/resources/web/js/message_parsing.mjs

```diff
@@ -127,7 +328,7 @@ export function initializeObfuscation() {
         // Uses requestAnimationFrame with timestamp checking for efficient rate limiting
         // that automatically pauses when tab is inactive
         if (timestamp - lastUpdate >= updateInterval) {
-            const elements = document.getElementsByClassName('minecraft-obfuscated');
+            const elements = document.getElementsByClassName('mc-obfuscated');
```

Comment:
I threw in this random change because the different class name format was upsetting me.